src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -198,41 +203,52 @@ export default {
 			return ''
 		},
 
+		updateCollapsedState() {
+			for (const group of this.messagesGroupedBySystemMessage) {
+				const isLastReadInsideGroup = this.lastReadMessageId >= group.id && this.lastReadMessageId < group.lastId
+				if (isLastReadInsideGroup) {
+					// If the last read message is inside the group, we should show the group expanded
+					group.collapsed = false
+				} else if (this.groupIsCollapsed[group.id] !== undefined) {
+					// If the group was collapsed before, we should keep it collapsed
+					group.collapsed = this.groupIsCollapsed[group.id]
+				} else {
+					// If the group is not collapsed, we should collapse it if it contains more than one message
+					group.collapsed = group.messages.length > 1
+				}
+			}
+		},
+
 		groupMessages(messages) {
 			const groups = []
 			let lastMessage = null
 			let forceNextGroup = false
 			for (const message of messages) {
 				const groupingType = this.messagesShouldBeGrouped(message, lastMessage)
 				if (!groupingType || forceNextGroup) {
-					groups.push({ id: message.id, lastId: message.id, messages: [message], type: '', collapsed: this.groupIsCollapsed[message.id] ?? true })
+					// Adding a new group
+					groups.push({ id: message.id, lastId: message.id, messages: [message], type: '', collapsed: group.messages.length > 1 })
```

Comment:
`ReferenceError: group is not defined`. Even if it will be, length here is 1, so doesn't make sense. Just omit it, as you going to iterate over all groups anyway 

```suggestion
					groups.push({ id: message.id, lastId: message.id, messages: [message], type: '' })
```

Reply:
I kept it as a hint for the structure of the object.